sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -447,15 +433,14 @@ public synchronized void onStarted(@NonNull Context context, @NonNull Channel ch
      * Set context, used when need to manipulate context before onStarted.
      * For example when download completes after application process exited.
      */
-    synchronized ReleaseDetails startFromBackground(Context context) {
+    synchronized void startFromBackground(Context context) {
```

Comment:
To test this, kill the app process while browser is counting down before redirecting back.

Reply:
Actually no. On this stage `DeepLinkActivity` is used and `startFromBackground` isn't called.

To test this, kill the app process during the loading via DownloadManager.
Anyway, I've tested tried to kill the app in both cases and it works as expected